plugins/inventory/vmware_vm_inventory.py

```diff
@@ -395,211 +485,255 @@ def parse(self, inventory, loader, path, cache=True):
         # set _options from config data
         self._consume_options(config_data)
 
+        username = self.get_option('username')
```

Comment:
I am not sure if someone actually encrypts `username` or not. Not a blocker but just my opinion.

Reply:
Me :sweat_smile:,  but not always! 
Anyway, with ansible/ansible#67823 or similar PR will render this useless.